### PR TITLE
feat: Created model package & Task class with 'tasks' table

### PR DESCRIPTION
**Pull request by Trntybth**

### CODE REVIEW

File: pom.xml
Comment (AlanDavison) [inline pom.xml:1]: Good idea adding Lombok!

File: src/main/java/com/northcoders/gamboge/waddl_api/model/Task.java
Comment (AlanDavison) [inline src/main/java/com/northcoders/gamboge/waddl_api/model/Task.java:1]: Looks good!

**Review by AlanDavison — APPROVED** [review]

File: src/main/java/com/northcoders/gamboge/waddl_api/model/Task.java
Comment (Xhinsu) [reply src/main/java/com/northcoders/gamboge/waddl_api/model/Task.java:1]: looks good

File: pom.xml
Comment (Xhinsu) [reply pom.xml:1]: looks good

**Review by Xhinsu — APPROVED** [review]

**Pull request merged**